cypress/data/data.ts

```diff
@@ -5,13 +5,17 @@ export const env = {
 export const users = {
   monteTeacher: {
     username: "teacher@test.com",
-    password: "Eifi6eeChu",
+    password: "",
```

Comment:
We don't need the password key-pair here at all. Consider deleting it and type the password manually.

Reply:
you're right - I'll delete it
and there will be an option to enter manually